new(minutes): meeting notes 29th April 2020

fntlnz opened the pull request:
Here the meeting minutes for April 29th.

This is still missing the youtube link since I wasn't able to login into the CNCF zoom to grab it.

I'll try to get some help on that side and update once I have it.


Signed-off-by: Lorenzo Fontana <lo@linux.com>

poiana commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **fntlnz**
You can assign the PR to them by writing `/assign @fntlnz` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=falcosecurity%2Fcommunity).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/falcosecurity/community/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["fntlnz"]} -->

leodido commented:
Hey, @kris-nova can you help us with the recording link/file?

leogr commented on meeting-notes/2020-04-30.md:
```diff
@@ -0,0 +1,66 @@
+
+[More Information](https://github.com/falcosecurity/community)
+
+[Zoom Link](https://sysdig.zoom.us/my/falco)
```
This is not yesterday's link.

fntlnz commented:
I was able to read the content of that email and to grab the video.
I'm now uploading it on YouTube!

fntlnz commented:
Added the link to the recording  and fixed the date since it was the 29th and not the 30th.

The branch name is wrong but not a big deal rn.

PTAL @leodido @leogr @kris-nova

poiana commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/falcosecurity/community/pull/92#pullrequestreview-404162633" title="Approved">leodido</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=falcosecurity%2Fcommunity).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/falcosecurity/community/blob/master/OWNERS)~~ [leodido]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

poiana commented:
LGTM label has been added.  <details>Git tree hash: fdd4614d24c7e6136abef75cbf2c84d3660b9d1a</details>
